src/models/ResizedImages.ts

```diff
@@ -0,0 +1,14 @@
+import { getModelForClass, prop } from '@typegoose/typegoose'
+
+class ResizedImages {
+  @prop({ index: true, required: true })
+  cid!: string
+  @prop()
+  height!: number | null
```

Comment:
`height?: number` ?

Reply:
You can leave like this ofc

Reply:
MongoDB didn't like taking `undefined` for the `findOne` query, so I used an explicit null instead.